packages/office-addin-manifest/README.md

```diff
@@ -52,3 +53,13 @@ This value is the `<DisplayName>` element of `<OfficeApp>`.
 For more info, see [OfficeApp documentation](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/manifest/officeapp).
 
 #
+
+### validate 
+Determines whether the Office Add-in manifest is valid.
+
+Syntax:
+
+`office addin-manifest validate <manifest>`
```

Comment:
@akrantz Is this command missing the hyphen after `office` (in `office addin-manifest`)?  If yes, then please fix the same typo in the other two commands in the README (for `info` and `modify`).

Reply:
yes, fixed.